### PR TITLE
Documentation updates for 2.0

### PR DESCRIPTION
**Pull request by rakillen**
** This does not include Release Notes changes - those are being handled separately **

### CODE REVIEW

File: documentation/2.0/content/release-notes.md [hunk 1]
```diff
@@ -35,3 +35,9 @@ java.lang.IllegalArgumentException: In production mode, it's not allowed to set
 **ISSUE**: For WLS versions prior to 14.1.1, there is a problem setting the `GroupMemberOf` attribute in the `topology/Security/User` folder. The value is not persisted correctly, and the assignment will not be present when the domain is started.
 
 **ACTION**: Contact Oracle Support to obtain the patch for bug number 30319071 for your WebLogic Server version before running the tool.
+
+#### Problems setting `RotateLogOnStartup` attribute
+
+**ISSUE**: For WLS versions up to and including 14.1.2, there is a problem setting the `RotateLogOnStartup` attribute in various log file folders. The value is not persisted correctly, and the assignment will not be present when the domain is started.
```
Comment (robertpatrick) [inline documentation/2.0/content/release-notes.md:41]: I don't think we should mention 14.1.2.  The numbering for the next WebLogic release is still undecided (and we shouldn't be discussing future WLS releases anyway)...

Comment (rosemarymarano) [inline documentation/2.0/content/release-notes.md:41]: WLS 14.1.2 ? -> As far as I know, this release doesn't exist yet. Am I mistaken? If not, then you should not mention it in the docs unless you expect WLS 14.1.2 to go out before WDT 2.0.

**Review by robertpatrick — CHANGES_REQUESTED** [review]

File: documentation/2.0/content/userguide/tools-config/domain_def.md [hunk 2]
```diff
@@ -142,6 +142,21 @@ weight: 2
 
  If there are any server groups in the custom template that should be targeted to managed servers, they should be specified in the `serverGroupsToTarget` attribute, similar to `MY_MAN_SVR` in the example above.
 
+ #### Using Compact Profile
```
Comment (rosemarymarano) [inline documentation/2.0/content/userguide/tools-config/domain_def.md:145]: Using Compact Profile -> Using compact profile

File: documentation/2.0/content/userguide/tools-config/model_filters.md [hunk 3]
```diff
@@ -44,3 +44,22 @@ This example configures two filters for the Create Domain Tool: `fix-password.py
   ]
 }
 ```
+### Internal Filters
+
+Internal filters are provided by WebLogic Deploy Tooling to perform modifications to the domain model. They are referenced by ID in the target configuration file. This example shows use of the internal filter `wko_filter` and a user-defined filter `fix-password` for use with the Create Domain Tool.
```
Comment (rosemarymarano) [inline documentation/2.0/content/userguide/tools-config/model_filters.md:49]: shows use -> shows the use

File: documentation/2.0/content/userguide/tools-config/model_filters.md [hunk 4]
```diff
@@ -44,3 +44,22 @@ This example configures two filters for the Create Domain Tool: `fix-password.py
...
+```json
+{
+  "create": [
+    { "name": "fixPassword", "path": "/home/user/fix-password.py" },
+    { "id": "wko_filter" }
+  ]
+}
+```
+Internal filters cannot be copied modified like other types of filters.
```
Comment (rosemarymarano) [inline documentation/2.0/content/userguide/tools-config/model_filters.md:58]: copied modified -> copied or modified

File: documentation/2.0/content/userguide/tools-config/model_filters.md [hunk 5]
```diff
@@ -44,3 +44,22 @@ This example configures two filters for the Create Domain Tool: `fix-password.py
...
+    { "id": "wko_filter" }
+  ]
+}
+```
+Internal filters cannot be copied modified like other types of filters.  
+
+#### Internal filter `wko_filter`
+
+This filter prepares the domain model for use with WebLogic Kubernetes Operator. It performs the following modifications:
+- Remove any attributes that can only be deployed in online mode
```
Comment (rosemarymarano) [inline documentation/2.0/content/userguide/tools-config/model_filters.md:63]: Remove  -> Removes
can only be deployed in online mode -> can be deployed only in online mode

File: documentation/2.0/content/userguide/tools-config/model_filters.md [hunk 6]
```diff
@@ -44,3 +44,22 @@ This example configures two filters for the Create Domain Tool: `fix-password.py
...
+- Set the `CalculatedListenPorts` attribute to false for dynamic clusters
```
Comment (rosemarymarano) [inline documentation/2.0/content/userguide/tools-config/model_filters.md:64]: Set  -> Sets

File: documentation/2.0/content/userguide/tools-config/model_filters.md [hunk 7]
```diff
@@ -44,3 +44,22 @@ This example configures two filters for the Create Domain Tool: `fix-password.py
...
+- Warn if servers in a static cluster have different listen ports
```
Comment (rosemarymarano) [inline documentation/2.0/content/userguide/tools-config/model_filters.md:65]: Warn  -> Warns

**Review by rosemarymarano — COMMENTED** [review]
few edits

File: documentation/2.0/content/release-notes.md (hunk 1 above)
Comment (rakillen) [reply documentation/2.0/content/release-notes.md]: Changed this to "For existing WLS versions"

**Review by robertpatrick — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**